tools/report-converter/tests/unit/ubsan_output_test_files/ubsan2.cpp

```diff
@@ -0,0 +1,31 @@
+#include <limits.h>
```

Comment:
License text is missing

Reply:
We do not put license text in test source files.